modules/virt-configuring-vm-with-node-exporter-service.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-configuring-vm-with-node-exporter-service_{context}"]
+= Configuring a virtual machine with the node exporter service
+
+Download the `node-exporter` file on to the virtual machine and create a `systemd` service that runs the `node-exporter` service when the virtual machine boots.
+
+.Prerequisites
+. The pods for the component are running in the `openshift-user-workload-monitoring` project.
+. Grant the `monitoring-edit` role to users who need to monitor this user-defined project.
+
+.Procedure
+
+. Log on to the virtual machine.
+
+. Download the `node-exporter` file on to a virtual machine by using the directory path that applies to the version of `node-exporter` file.
++
+[source,terminal]
+----
+$ wget https://github.com/prometheus/node_exporter/releases/download/v1.3.1/node_exporter-1.3.1.linux-amd64.tar.gz
+----
+
+. Extract the executable and place it in the `/usr/bin` directory.
++
+[source,terminal]
+----
+$ sudo tar xvf node_exporter-1.3.1.linux-amd64.tar.gz \
+    --directory /usr/bin --strip 1 "*/node_exporter"
+----
+
+. Create a `node_exporter.service` file in this directory path: `/etc/systemd/system`. This `systemd` service file runs the `node-exporter` service when the virtual machine is rebooted.
++
+[source,terminal]
+----
+[Unit]
+Description=Prometheus Metrics Exporter
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=root
+ExecStart=/usr/bin/node_exporter
+
+[Install]
+WantedBy=multi-user.target
+----
+
+. Enable and start the `systemd` service.
++
+[source,terminal]
+----
+$ sudo systemctl enable metrics
+$ sudo systemctl start metrics
+----
+
+.Verification
+* Verify that the `node-exporter` agent is reporting metrics from the virtual machine.
+----
+$ curl http://localhost:9100/metrics
+----
+.Example output
+[source,terminal]
+----
+# HELP app_connections number of connections for an app
```

Comment:
These 4 app_connection lines wont be shown unless you create the custom monitor. They were defined by me. Not sure if you want to remove them or leave them in since it is just example output.

Reply:
Removed them since they're comments.